### PR TITLE
Add instruction for sidekiq prometheus monitoring

### PR DESCRIPTION
**Pull request by keiko713**
This was introduced with enterprise-v2023.03.0 release.
Documenting here so that the Enterprise customers could take advantage of it potentially.

### CODE REVIEW

File: enterprise/settings.mdx [hunk 1]
```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
 ```
 
 Note you can run an [Enterprise self-check](/docs/enterprise/troubleshooting) to verify the connection and lookup LDAP information, but some configuration errors are only visible once logging in.
+
+## Sidekiq monitoring using Prometheus
+
+The metrics HTTP endpoint `/metrics` is available for the Prometheus scraping. Currently, it provides Sidekiq metrics using [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
```
Comment (lfittl) [inline enterprise/settings.mdx:134]: From a grammatical perspective, I think the gem should be prefixed with "the". Separately, I think there should be no "the" prefixing Prometheus.

Maybe we can rewrite like this:

```suggestion
The metrics HTTP endpoint `/metrics` is available for Prometheus scraping. Currently, it provides worker queue metrics using the [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
```

File: enterprise/settings.mdx [hunk 2]
```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
 ```
 
 Note you can run an [Enterprise self-check](/docs/enterprise/troubleshooting) to verify the connection and lookup LDAP information, but some configuration errors are only visible once logging in.
+
+## Sidekiq monitoring using Prometheus
```
Comment (lfittl) [inline enterprise/settings.mdx:132]: Maybe "Worker queue monitoring using Prometheus"?

(the fact that we're using Sidekiq is an implementation detail in a sense)

File: enterprise/settings.mdx [hunk 3]
```diff
@@ -128,3 +128,23 @@ DEFAULT_ORG_ROLE=view_all
...
+
+The metrics HTTP endpoint `/metrics` is available for the Prometheus scraping. Currently, it provides Sidekiq metrics using [sidekiq-prometheus-exporter gem](https://github.com/Strech/sidekiq-prometheus-exporter).
+
+The endpoint is protected with a bearer auth, the token is specified with the `SIDEKIQ_PROMETHEUS_TOKEN` environment variable. The endpoint is only available when the environment variable is set.
+If you have a Prometheus setup already, you can add the hostname and port of your web server as an additional target to the prometheus config file to scrape.
```
Comment (lfittl) [inline enterprise/settings.mdx:137]: For consistency, we should probably uppercase Prometheus in the later part of the sentence too:

```suggestion
If you have a Prometheus setup already, you can add the hostname and port of your web server as an additional target to the Prometheus config file to scrape.
```

**Review by lfittl — APPROVED** [review]
Thanks for adding this!

Some minor notes, but looks good overall. 🚢

File: enterprise/settings.mdx (hunk 2 above)
Comment (keiko713) [reply enterprise/settings.mdx:132]:
```suggestion
## Worker queue monitoring using Prometheus
```

**Pull request merged**